Client/Extensions/UIImageViewExtensions.swift

```diff
@@ -73,6 +87,14 @@ public extension UIImageView {
             return (FaviconFetcher.defaultFavicon, .white)
         }
     }
+    
+    private func defaultIconIfNeeded(_ url: URL?) -> (image: UIImage, color: UIColor)? {
+        if let image = image, let url = url, image.size.width <= 57 || image.size.height <= 57 {
```

Comment:
Use `31`, as 32x32 favicon often looks ok:

<img width="196" alt="image" src="https://user-images.githubusercontent.com/5495083/70740059-50179500-1ce6-11ea-8ae4-45e1a5358843.png">

Ideally we wouldn't scale 32x32 up so much.

Reply:
Applied changes